### PR TITLE
T2D-184 (PART 2) POM Submodule generation

### PR DESCRIPTION
**Pull request by jorizci**

### CODE REVIEW

File: accession-commons-monotonic-generator-jpa/src/test/java/uk/ac/ebi/ampt2d/commons/accession/core/BasicMonotonicAccessioningTest.java [hunk 1]
```diff
@@ -114,7 +116,7 @@ public void testGetWithExistingEntries() throws AccessionCouldNotBeGeneratedExce
                         TestModel.of("service-test-3")
                 ));
         assertEquals(1, accessions2.size());
-        assertEquals("service-test-3", accessions2.get(0).getData().getValue());
+        Assert.assertEquals("service-test-3", accessions2.get(0).getData().getValue());
```
Comment (jmmut) [inline accession-commons-monotonic-generator-jpa/src/test/java/uk/ac/ebi/ampt2d/commons/accession/core/BasicMonotonicAccessioningTest.java]: what? why?

File: accession-commons-jpa/src/main/java/uk/ac/ebi/ampt2d/commons/accession/persistence/jpa/accession/entities/InactiveAccessionEntity.java [hunk 2]
```diff
@@ -18,6 +18,7 @@
 package uk.ac.ebi.ampt2d.commons.accession.persistence.jpa.accession.entities;
```
Comment (selvaebi) [inline accession-commons-jpa/src/main/java/uk/ac/ebi/ampt2d/commons/accession/persistence/jpa/accession/entities/InactiveAccessionEntity.java]: can this be uk.ac.ebi.ampt2d.commons.accession.persistence.jpa.entities ??

Comment (jorizci) [reply accession-commons-jpa/src/main/java/uk/ac/ebi/ampt2d/commons/accession/persistence/jpa/accession/entities/InactiveAccessionEntity.java]: It is possible, @cyenyxe your take?

Comment (cyenyxe) [reply accession-commons-jpa/src/main/java/uk/ac/ebi/ampt2d/commons/accession/persistence/jpa/accession/entities/InactiveAccessionEntity.java]: Agree

**Pull request merged**